.github/ISSUE_TEMPLATE/venue-host.md

```diff
@@ -2,7 +2,7 @@
 name: Host request
 about: Host an upcoming VanJS event at your venue
 title: 'Host request: '
-labels: Host
```

Comment:
I'm not sure why it's showing changes to these files... it must have automatically updated them?

Reply:
looks like it removed it since the label name was in correct. Should be correct now